source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs

```diff
@@ -492,6 +492,22 @@ private static void LogUsage()
         {
             Logger.d("Received method call on DummyClient - using stub implementation.");
         }
+
+        /// <summary>
+        /// Shows the Player Games Profile UI for the given player Id.
+        /// </summary>
+        /// <param name="playerId">Player Id. This cannot be null.</param>
+        /// <param name="callback">Callback to invoke when complete.</param>
+        public void ShowPlayerProfileUI(
```

Comment:
It seems it is using CompareProfileIntent, so name should be ShowCompareProfileUI in this case